packages/ocean-react/src/Chart/utils/chartUtils.ts

```diff
@@ -0,0 +1,45 @@
+export const options = {
```

Comment:
These options const could be in de component file, they are not that long and have essential information about the component behavior.